notes.md

```diff
@@ -3,27 +3,33 @@
 
 # Beginning of File
 
-An EBML file always starts with 0x1A. The 0x1A makes the DOS command "type" ends display. That way you can include ASCII text before the EBML data and it can be displayed. The EBML parser is safe from false-alarm with these ASCII only codes.
+An EBML file MUST start with `0x1A`. The 0x1A makes the DOS command "type" ends display. Therefore, ASCII text can be included before the EBML data and it can be displayed. After `0x1A`, the EBML header is stored. This allows the the parser to know what type of EBML file it is parsing.
```

Comment:
Is this section necessary for the specification? Can it be remove as "out of scope"?

Reply:
At least
>An EBML file MUST start with `0x1A`.

is necessary here, in my opinion. Yet I think more does not hurt.

Reply:
To follow this up, if it should be here, I think it should move somewhere, possibly the top of the Ordered Guidelines section.

Reply:
This should be reviewed carefully as removing it would invalidate potential files in the wild. Personally I am not supportive of the ASCII before EBML header idea. From this sentence it seems like having 10 gigabytes of 0x20 bytes followed by a Matroska file would be a valid Matroska file. If this section is kept I'd suggest it be limited a reasonable number of bytes. I think MXF is a bit similar where it can start with junk before the header but I hope they apply a limit to it.

[after some testing]. I tried appending a 0x20 to the beginning of Matroska and then it fails to play in VLC or ffmpeg. @robUx4 is there a particular use case planned with this ASCII feature or could we drop this?

Reply:
My interest in removal has to do with it being a rule for EBML (and moved to that spec, if not already written or otherwise stood there), and not Matroska specifically.

Reply:
If kept (or moved), maybe language should change from "can be included" to SHOULD NOT -- dictating that it would not technically be invalid if done, but is not recommended.

Reply:
I'd vote no on moving it to EBML. This feature was historically only listed in Matroska specs and not referenced in EBML. I'd prefer not to allow this feature to infect other EBML based formats.

Reply:
Definitely not move it to EBML, please. I'd rather just remove that section. I've never seen such a file in the wild (by which I don't mean that they don't exist, of course). Right now seems a good opportunity to remove such cruft.

Reply:
Resolved in https://github.com/Matroska-Org/matroska-specification/pull/177